sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentPageContentService.java

```diff
@@ -94,4 +96,16 @@ private JsonObject getExperimentFileAsJson(ExperimentFileType experimentFileType
         return result;
     }
 
+    private JsonObject getDownloadSection(String sectionName, List<ExperimentFileType> experimentFileTypes, String experimentAccession, String accessKey) {
```

Comment:
I applaud your decision of not nameing this method as `getDownloadSectionAsJson`!

Reply:
Would you like me to remove `asJson` from all the other method names in this class? 😃